src/modules/ticket/actionCreators.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  BuyTicketFailAction,
+  BuyTicketRequestAction,
+  BuyTicketSuccessAction,
+} from './reducer';
+
+export const buyTicketRequest = (flightId: string): BuyTicketRequestAction => {
+  return {
+    type: 'BUY_TICKET_REQUEST',
+    payload: flightId,
+  };
+};
+
+export const buyTicketSuccess = (
+  reservationId: string,
+): BuyTicketSuccessAction => {
+  return {
+    type: 'BUY_TICKET_SUCCESS',
+    payload: {reservationId},
+  };
+};
+
+export const buyTicketFail = (error: string): BuyTicketFailAction => {
+  return {
+    type: 'BUY_TICKET_FAIL',
```

Comment:
Line:9, Line:18, Line:25, Should be fine if an `ActionConstant` is created.
 > Constant variables on `Line:5` to `Line:7` should be moved to a separate file inside `modules/ticket`. It an be called something like `ActionConstant.ts`
  https://github.com/adescode/CRTASK/pull/1#discussion_r865464051